v2/blockstore/ro_blockstore.go

```diff
@@ -0,0 +1,190 @@
+package blockstore
+
+import (
+	"bufio"
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"golang.org/x/exp/mmap"
+	"io"
+
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	carv1 "github.com/ipld/go-car"
+	"github.com/ipld/go-car/util"
+	"github.com/ipld/go-car/v2/internal/index"
+	internalio "github.com/ipld/go-car/v2/internal/io"
+)
+
+var _ blockstore.Blockstore = (*ReadOnlyBlockStore)(nil)
+
+// errUnsupported is returned for unsupported operations
+var errUnsupported = errors.New("unsupported operation")
+
+type (
```

Comment:
any particular reason you use `()` groups for single types? consistency? it does add two lines for very little benefit, so I'd probably avoid it.

Reply:
face of habit I usually do this to group type declarations